README.md

```diff
@@ -28,6 +28,11 @@
   &nbsp;&nbsp;&nbsp;
   <a href="https://join.slack.com/t/psgdataset/shared_invite/zt-1d14sdkw3-59pJdrp6gLAHuBObPL91qw" target='_blank'>
     <img src="https://img.shields.io/badge/Forum-Slack-4c1448?style=flat-square">
+    &nbsp;&nbsp;&nbsp;
+  <a href="https://replicate.com/cjwbw/openpsg" target='_blank'>
+    <img src="https://img.shields.io/badge/Replicate-Demo & Cloud API-1b82c2?style=flat-square">
+
+    [![Replicate](https://replicate.com/kimgeonung/bigcolor/badge)](https://replicate.com/kimgeonung/bigcolor)
```

Comment:
Hi! What is this line for?

Reply:
Ah sorry was accidental (that was our usual badge so I was copy pasting for the new url but then realised you have a different template but forgot to remove the old one) - have removed now!